_topic_maps/_topic_map_osd.yml

```diff
@@ -251,6 +251,83 @@ Topics:
     - Name: Configuring custom domains for applications
       File: osd-config-custom-domains-applications
 ---
+Name: Logging
+Dir: logging
+Distros: openshift-dedicated
+Topics:
+- Name: Release notes
+  File: cluster-logging-release-notes
+- Name: About Logging
+  File: cluster-logging
+- Name: Installing AWS CloudWatch logging add-on service
```

Comment:
Should this one be in OSD? If so, it'll need some conditionals (currently rendering as ROSA): https://53099--docspreview.netlify.app/openshift-dedicated/latest/logging/sd-install-cloudwatch-logging.html